src/Plugin/GraphQL/Fields/SearchAPIFacetValues.php

```diff
@@ -23,8 +23,12 @@ class SearchAPIFacetValues extends FieldPluginBase {
    * {@inheritdoc}
    */
   public function resolveValues($value, array $args, ResolveContext $context, ResolveInfo $info) {
-    foreach ($value['solrFacetValues'] as $facet_value) {
-      yield $facet_value;
+    $key = 'solrFacetValues';
```

Comment:
I think here we could actually just have :

```suggestion
if (isset($value['solrFacetValues'])) {
    foreach ($value['solrFacetValues'] as $facet_value) {
        yield $facet_value;
    }
}
```
What do you think? sounds a bit clearer to me.

Reply:
If you agree we can commit this suggestion and merge it ;)

Reply:
It is not a big deal to me, because the key value is referenced more than once, I prefer to put the value into a variable, similar to using constants